### DISCUSSION TITLE
Queue Manage: "Update of Products with inconsistent Attributes" Endless Loop

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#284] [Q&A] [answered]
Not sure how to resolve this and what causes it.

![image](https://user-images.githubusercontent.com/1337223/152605500-d332d627-4dda-448d-9444-dfd41bd67d37.png)

![image](https://user-images.githubusercontent.com/1337223/152605768-bd8841b5-63b3-497d-930e-fb0b84d90feb.png)

Also blocks the Update

![image](https://user-images.githubusercontent.com/1337223/152605906-a2bf1515-7317-4bbd-974f-3b3bf6d30da6.png)

**Comment by atrolex** [comment, marked as answer]
Can you describe, what exactly have you done? 
.. actually we do refactor this algorithm now, we will replace it with "pseudo transactions" hoping that it will work more stable. It is already work in progress.

**Reply by marcus-at-localhost** [reply]
Unfortunately I don't remember what I did - I just noticed it at some point that this process was running.
I looked at the source code and tried to figure out what was happening when the system checks for inconsistent attributes, but couldn't figure it out. So I asked here.

**Reply by marcus-at-localhost** [reply]
This was fixed with some update! Thank you, what ever you did :)
